filter.go

```diff
@@ -166,6 +178,12 @@ func compileFilter(filter string, pos int) (*ber.Packet, int, error) {
 		}
 	}()
 
+	xpar_stop , _ := utf8.DecodeRuneInString(")")
```

Comment:
If so, keeping the cast literals inline below would be easier to follow, I think